frontend/src/components/grantaccess/form/UserNameInput.vue

```diff
@@ -28,25 +32,50 @@ const computedUserId = computed({
     },
 });
 
-const verifiedUserIdentity = ref<IdimProxyIdirInfo | null>(null);
+const errorMgs = ref('');
```

Comment:
Hi Jota, do you mean "errorMsg" or you really mean "Mgs"? (no need to change if you mean Mgs)

Reply:
I changed it